docs/program/extend/modular-resources/examples/add-tflite-module.md

```diff
@@ -0,0 +1,232 @@
+- [`example_audio_classification_client.cpp`](https://github.com/viamrobotics/viam-cpp-sdk/blob/main/src/viam/examples/mlmodel/example_audio_classification_client.cpp) - an example client instance which generates audio samples and invokes the `example_mlmodelservice_tflite` custom module to classify those samples.
+
+This tutorial walks you through everything necessary to start using these example files with your robot, including building the C++ SDK, procuring the necessary support files, configuring your robot and installing `viam-server`, and running the compiled binary example.
+
+## Build the C++ SDK
+
+To build the Viam C++ SDK, you will need a macOS or Linux computer.
+Follow the instructions below for your platform:
+
+{{< tabs >}}
+{{% tab name="macOS" %}}
+
+Follow the [Viam C++ SDK build instructions](https://github.com/viamrobotics/viam-cpp-sdk/blob/main/BUILDING.md) to build the SDK on your macOS computer using the `brew` package manager.
+
+While your specific build steps may differ slightly, your installation should generally resemble the following:
+
+1. Install all listed dependencies to support the Viam C++ SDK:
+
+   ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+   brew install abseil cmake boost grpc protobuf xtensor pkg-config ninja buf
+   ```
+
+1. Clone the Viam C++ SDK to your computer:
+
+   ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+   mkdir -p ~/example_workspace
+   cd ~/example_workspace
+   git clone git@github.com:viamrobotics/viam-cpp-sdk.git
+   ```
+
+1. Create a <file>build</file> directory to house the build target:
+
+   ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+   cd viam-cpp-sdk/
+   mkdir build
+   cd build
+   ```
+
+1. The Viam C++ SDK requires a specific version of `openssl` in order to build successfully.
+   Determine the version of `openssl` you are running on your macOS computer:
+
+   ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+   brew list | grep openssl
+   ```
+
+   If the command returns `openssl@1.1`, proceed to the next step. If the command returns a different `openssl` version, install version `openssl@1.1` specifically:
+
+   ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+   brew install openssl@1.1
+   ```
+
+1. Create an environment variable `PKG_CONFIG_PATH` which points to the version of `openssl` we will use for the Viam C++ SDK build:
+
+   ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+   export PKG_CONFIG_PATH="/opt/homebrew/opt/openssl@1.1/lib/pkgconfig"
+   ```
+
+1. Build the C++ SDK by running the following commands:
+
+   ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+   cmake .. -DVIAMCPPSDK_USE_DYNAMIC_PROTOS=ON -DCMAKE_INSTALL_PREFIX=/usr/local -G Ninja
```

Comment:
I'm not sure it is a good idea to install to `/usr/local`. That's a "real" location on some systems and requires privs to write to, per the `sudo` used below. Maybe `$HOME/opt` is a better default choice. Maybe some language identifying that an installation prefix is suggested but that by default it will just install to the `install` subdirectory of your build.

Or, even better maybe, have it install to `~/example_workspace/opt` or something, since you have already created that location?

Reply:
Thank you -- great point. I've converted both the macOS and Linux instructions to use `~/example_workspace/opt` instead. 👍